unit-13/javascrip-even/main.js

```diff
@@ -0,0 +1,47 @@
+// lesson 1
+const date = new Date()
+const day = date.getDate()
+const month = date.getMonth() + 1
+const year = date.getFullYear()
+const hours = date.getHours()
+const minutes = date.getMinutes()
+const seconds = date.getSeconds()
+const currentTime = `${day}-${month}-${year} ${hours}:${minutes}:${seconds}`
+
+document.getElementById('onmouse').onmouseover = function () {
+	console.log(`Hover lúc: ${currentTime}`)
+}
+document.getElementById('onmouse').onmouseout = function () {
+	console.log(`Out hover lúc: ${currentTime}`)
+}
+// lesson 2
+document.getElementById('input').oninput = function () {
+	const inputValue = document.getElementById('input').value
+	const regex = /^[a-z0-9_]{0,}[^A-Z\.,!@#$%^&*()~+-][a-z0-9_]{0,}$/
+	if (!(regex.test(inputValue))) {
+		alert('Date a-z (lowercase), and the numbers 0-9, and an underscore')
+	}
+}
+// lesson 3
```

Comment:
Bài tập số 3, chứ ko phải bài học số 3
```suggestion
// Lesson 3
```
Và nếu viết là `lesson` thì hãy viết hoa chữ `L` đi